packages/app-mobile/commands/newNote.ts

```diff
@@ -0,0 +1,32 @@
+import { CommandRuntime, CommandDeclaration, CommandContext } from '@joplin/lib/services/CommandService';
+import Logger from '@joplin/utils/Logger';
+import goToNote from './util/goToNote';
+import Note from '@joplin/lib/models/Note';
+import Setting from '@joplin/lib/models/Setting';
+
+const logger = Logger.create('newNoteCommand');
+
+export const declaration: CommandDeclaration = {
+	name: 'newNote',
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async (_context: CommandContext, body = '', todo = false) => {
+			const folderId = Setting.value('activeFolderId');
+			if (!folderId) {
+				logger.warn('Not creating new note -- no active folder ID.');
```

Comment:
It could make sense to update the "new note" and "new to-do" buttons to use this command.

However, the "new note"/"to-do" buttons currently `alert` if creating a note fails. This command just logs.